packages/grafana-ui/src/components/PanelChrome/PanelChrome.tsx

```diff
@@ -289,6 +306,7 @@ const getStyles = (theme: GrafanaTheme2) => {
       display: 'flex',
       alignItems: 'center',
       overflow: 'hidden',
+      height: '100%',
```

Comment:
I forgot to add this to the review 🙈 , here the `height: '100%',` it is also breaking the focusRing style

https://user-images.githubusercontent.com/239999/214545074-6c6565de-8b5e-4f74-add5-527a58160775.mp4